magento/pwa-studio
# Search RootComponent -> SearchPage Component + Talon

## Description

We've added talons for most of the components in `venia-ui`, but not for some of the root component. Create a talon for `Search`:

Create a `SearchPage` directory and `useSearchPage.js` in `peregrine/lib/talons`
Create a `SearchPage` component in `venia-ui/lib/components` and use the talon
Rewrite the `Search` root component to be a mere shell around `SearchPage`

## Related Issue





Closes [PWA-87](https://jira.corp.magento.com/browse/PWA-87) Create a talon for the Search root component

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to Search page (`/search.html?query=shirt`)
2. Verify Jillian Top is displayed
3. Go wild, verify that no regressions were introduced

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
